### DISCUSSION TITLE
How to move block manager only left side?

### DISCUSSION THREAD

**Discussion by san-123** [GrapesJS/grapesjs#3983] [General]
Hi @artf ,

Hi I need to move block manager left side and other are right side.

Any idea, please guide me.

Thanks

**Comment by Ju99ernaut** [comment]
You can create custom UI as shown [here](https://codepen.io/ju99ernaut/pen/WNoOWZy)

**Reply by san-123** [reply]
@Ju99ernaut,

Thanks works fine,  one more clarification, I have more than 2blocks each block more than 10 contros,  so how to show single blocks at a time?
